src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationServiceTest.java

```diff
@@ -42,7 +43,15 @@ public void setup() {
 
     @Test
     public void notifyClaimantShouldCallNotify() throws Exception {
-        Claim claim = SampleClaim.getDefault();
+        Claim claim = SampleClaim.builder()
+            .withClaimData(
+                SampleClaimData.builder()
```

Comment:
but you probably don't need to add specific defendant, do you?

Reply:
yes, agree, will remove that